### DISCUSSION TITLE
Is some option how install A11 GSI on phone with smaller partition?

### DISCUSSION THREAD

**Discussion by Axident1973** [phhusson/treble_experimentations#1695] [General]
Hi, I install A10 GSI from you on cheap Vernee T3 Pro and working like a charm, but A11 need more space on partition, exist some way, how resize right partition without TWRP? Thanks,

**Comment by martinfutoo** [comment]
Try again with the latest build. It's smaller. I had the same issue with Xiaomi Redmi Go. Using A10 OK. A11 too big. Now system image fits, but it reboots into recovery every time. So it's restore from backup and try again next build. Good luck.

**Comment by Axident1973** [comment]
Thank you very much, i will try...

Sent from Yahoo Mail on Android 
 
  On st, bře 10, 2021 at 11:15, martinfutoo<notifications@github.com> wrote:   


Try again with the latest build. It's smaller. I had the same issue with Xiaomi Redmi Go. Using A10 OK. A11 too big. Now system image fits, but it reboots into recovery every time. So it's restore from backup and try again next build. Good luck.
